src/ng/parse.js

```diff
@@ -1703,11 +1703,11 @@ function $ParseProvider() {
 
   this.$get = ['$filter', '$sniffer', function($filter, $sniffer) {
     var $parseOptions = {
-          csp: $sniffer.csp,
+          csp: $sniffer.csp && $sniffer.csp.noUnsafeEval,
```

Comment:
Actually, perhaps we don't need to use `$sniffer` here at all? Since we are inside the angular.js closure we could just use the `csp()` function. I don't think we are mocking it anywhere are we?